namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-person-record-prod/resources/s3-ndelius-extract.tf

```diff
@@ -0,0 +1,76 @@
+module "hmpps-person-record-ndelius-s3-extract" {
+
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=5.1.0"
+  team_name              = var.team_name
+  business_unit          = var.business_unit
+  application            = var.application
+  is_production          = var.is_production
+  namespace              = var.namespace
+  environment_name       = var.environment
+  infrastructure_support = var.infrastructure_support
+
+  versioning = false
+
+  providers = {
+    aws = aws.london
+  }
+
+
+  bucket_policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "S3PutPolicy",
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::050243167760:root"
+            },
+            "Action": [
+                "s3:PutObject",
+                "s3:PutObjectAcl"
+            ],
+            "Resource": [
+                "$${bucket_arn}/*",
+                "$${bucket_arn}"
+            ],
+            "Condition": {
+                "StringEquals": {
+                    "s3:x-amz-acl": "bucket-owner-full-control"
+                }
+            }
+        },
+        {
+             "Sid": "S3ListPolicy",
+             "Effect": "Allow",
+             "Principal": {
+                 "AWS": "arn:aws:iam::050243167760:root"
```

Comment:
if "The intention is to create a bucket which is only readable by the IRSA account named person-record-service" then why not use that role arn here?